itests/raft_messagesigner_test.go

```diff
@@ -536,7 +542,7 @@ func TestGoRPCAuth(t *testing.T) {
 	wrappedFullNode := kit.MergeFullNodes(nodes)
 
 	ens.MinerEnroll(&miner, wrappedFullNode, kit.WithAllSubsystems(), kit.ThroughRPC())
```

Comment:
If the below changes work well, could you also change this in the setup function?

Reply:
Done!